packages/console/src/pages/Connectors/components/ConnectorForm/index.tsx

```diff
@@ -25,6 +26,17 @@ const ConnectorForm = ({ connector }: Props) => {
     setDarkVisible((previous) => !previous);
   };
 
+  const syncProfileOptions = [
+    {
+      value: 'OnlyAtRegister',
+      title: t('connectors.guide.sync_profile_only_at_register'),
+    },
+    {
+      value: 'EachSignIn',
+      title: t('connectors.guide.sync_profile_each_sign_in'),
+    },
+  ];
```

Comment:
1. Better have an enum for those two options
2. Might be better to call it `OnInit` and `Always`?

Reply:
Enum added, keep the original name as corresponding to the content.